nf_research_tools.rdb.model.csv

```diff
@@ -55,15 +55,15 @@ Tumor Type,"The types of tumors that are banked, if applicable.","cutaneous neur
 Specimen Format,How the specimens have been processed in preparation for distribution.,"whole tumor, slide, whole cell lysate, DNA, RNA",,TRUE,,Biobank,,,list
-Observation,"A remark, statement, or comment based on the resource.",,"Component, Observation_id, Resource_id, Publication_id, Observation Submitter Name, Synapse_id, Observation Text, Observation Type, Observation Reference, Observation Time, Observation Time Units, Reliability Rating, Ease of Use Rating, Observation Link",FALSE,,Observation,"Observation_id, Resource_id, Publication_id, Observation Submitter Name, Synapse_id, Observation Text, Observation Type, Observation Reference, Observation Time, Observation Time Units, Reliability Rating, Ease of Use Rating, Observation Link","Resource, Investigator, Publication",
+Observation,"A remark, statement, or comment based on the resource.",,"Component, Observation_id, Resource_id, Publication_id, Observation Submitter Name, Synapse_id, Species, Observation Text, Observation Type, Observation Phase, Observation Time, Observation Time Units, Reliability Rating, Ease of Use Rating, Observation Link",FALSE,,Observation,"Observation_id, Resource_id, Publication_id, Observation Submitter Name, Synapse_id, Species, Observation Text, Observation Type, Observation Phase, Observation Time, Observation Time Units, Reliability Rating, Ease of Use Rating, Observation Link","Resource, Investigator, Publication",
 Observation_id,A unique identifier for the observation.,,,TRUE,,Observation,,,
 Observation Submitter Name,The name of the person submitting the observation. ,,,TRUE,,Observation,,,
 Synapse ID,The synapse identifier for the person submitting the observation. ,,,FALSE,,Observation,,,
 Observation Text,Free text observation about the resource.,,,TRUE,,Observation,,,
-Observation Type,Type of observation,"pathologic, behavioral-non pathologic, issue, neurochemical, Depositor Comment, Usage Instructions, general comment/review, other",,TRUE,,Observation,,,
-Observation Reference,What t=0 for the observationTime is,"tumor implantation, gestational day zero, birth, immortalization",,FALSE,,Observation,,,
+Observation Type,Type of observation,"Body Length, Body weight, Coat Color, Disease Susceptibility, Feed Intake, Feeding Behavior, Growth rate, Motor Activity, Organ Development, Reflex Development, Reproductive Behavior, Social Behavior, Swimming Behavior, Tumor Growth, Other",,TRUE,,Observation,,,
+Observation Phase,What t=0 for the observationTime is,"Adolescent, Adult, Death Phase, Embryo, Juvenile, Larval, Log Phase, Neonatal, Postnatal, Prenatal, Pupal, Stationary Phase, Weanling",,FALSE,,Observation,,,
 Observation Time,When the observation was.,,,FALSE,,Observation,,,
-Observation Time Units,The unit of time pertaining to the observation,"hour, day, week, month, year",,FALSE,,Observation,,,
+Observation Time Units,The unit of time pertaining to the observation,"E, hour, day, week, month, year",,FALSE,,Observation,,,

```

Comment:
```suggestion
Observation Time Units,The unit of time pertaining to the observation,"days, weeks, months, years, hours, minutes, seconds, milliseconds",,FALSE,,Observation,,,
```

Per Jay's comment here: https://sagebionetworks.jira.com/browse/NFINT-813?focusedCommentId=183191

I remember we discussed adding in "E", but since this is really a synonym for Days, and "Embryo" is captured in the Observation Phase, maybe we should not have a special value for this here?

Reply:
Gotcha - ok yeah that makes sense to just use Days